README.rst

```diff
@@ -76,18 +76,31 @@ than one CPU core, refer to our `parallel backend guide`_.
 
 **Note for Windows users**
 
-The pip installer for Neuron does not yet work for Windows. In this case, it is better to
-install ``hnn_core`` without the dependencies::
+The pip installer for Neuron does not yet work for Windows. In this case we recommend
+installing `WSL <https://docs.microsoft.com/en-us/windows/wsl/install>`_ on your
+local machine, and install ``hnn-core`` using the same steps as above.
 
-   $ pip install hnn_core --no-deps
-
-and then install the dependencies separately::
+Otherewise, to use ``hnn-core`` natively on windows, it is necessary to first install the
+dependencies separately::
 
    $ pip install scipy numpy matplotlib
 
-and install Neuron using the traditional package installer available here
+install Neuron using the traditional package installer available here
```

Comment:
I thought the pip installation for Neuron works, but it's just that the mod files need to be build by calling them manually?

Reply:
Oh wow I think you're right, that is actually new. Do you have a windows machine you can verify this with?

Reply:
I was thinking that it was possible all along, but that's probably my ignorance talking. I could test it out tonight, but it might be prudent to merge this PR before then.

Reply:
I don't think Neuron can be installed on Windows with pip ...